src/clj/controllers/question/question_controller.clj

```diff
@@ -76,8 +76,7 @@
                                           achivable-points
                                           type
                                           possible-solutions
-                                          single-choice-solution
-                                          multiple-choice-solution
+                                          (or single-choice-solution multiple-choice-solution)
```

Comment:
This was a little bit overeager, I wanted to fix this in the next PR.
But thanks ;)